### PR TITLE
New Rule: CVE-2024-21413 Microsoft Outlook Remote Code Execution

### PR DESCRIPTION
**Pull request by morriscode**

### CODE REVIEW

File: detection-rules/link_cve_2024-21413.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: "Link: CVE-2024-21413 Microsoft Outlook Remote Code Execution Vulnerability"
+description: "This rule detects messages containing links exploiting CVE-2024-21413. This attack can bypass built-in Outlook protections for malicious links embedded in emails using the file:// protocol and adding an exclamation mark to URLs pointing to attacker-controlled servers."
+references:
+  - "https://www.bleepingcomputer.com/news/security/new-critical-microsoft-outlook-rce-bug-is-trivial-to-exploit/"
+  - "https://msrc.microsoft.com/update-guide/en-US/advisory/CVE-2024-21413"
+type: "rule"
+severity: "high"
```
Comment (jkamdjou) [inline detection-rules/link_cve_2024-21413.yml:7]:
```suggestion
severity: "critical"
```
?

**Comment by morriscode** [comment]
FWIW, this should be super safe to deploy, it's an abnormal structure to begin with. I haven't been able to trigger any FP's with the existing logic.

File: detection-rules/link_cve_2024-21413.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+name: "Link: CVE-2024-21413 Microsoft Outlook Remote Code Execution Vulnerability"
+description: "This rule detects messages containing links exploiting CVE-2024-21413. This attack can bypass built-in Outlook protections for malicious links embedded in emails using the file:// protocol and adding an exclamation mark to URLs pointing to attacker-controlled servers."
```
Comment (jkamdjou) [inline detection-rules/link_cve_2024-21413.yml:2]:
```suggestion
description: "This rule detects messages containing links exploiting CVE-2024-21413, which can lead to RCE.

The attack can bypass built-in Outlook protections for malicious links embedded in messages by using the file:// protocol and an exclamation mark to URLs pointing to attacker-controlled servers."
```

File: detection-rules/link_cve_2024-21413.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+references:
```
Comment (jkamdjou) [inline detection-rules/link_cve_2024-21413.yml:3]:
```suggestion
references:
  - "https://research.checkpoint.com/2024/the-risks-of-the-monikerlink-bug-in-microsoft-outlook-and-the-big-picture/"
```

**Review by jkamdjou — APPROVED** [review]

File: detection-rules/link_cve_2024-21413.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+name: "Link: CVE-2024-21413 Microsoft Outlook Remote Code Execution Vulnerability"
+description: "This rule detects messages containing links exploiting CVE-2024-21413, which can lead to RCE.
+
+The attack can bypass built-in Outlook protections for malicious links embedded in messages by using the file:// protocol and an exclamation mark to URLs pointing to attacker-controlled servers."
```
Comment (jkamdjou) [inline detection-rules/link_cve_2024-21413.yml:4]:
```suggestion
description: |
  This rule detects messages containing links exploiting CVE-2024-21413, which can lead to RCE.

  Successful exploitation can bypass built-in Outlook protections for malicious links embedded in messages by using the file:// protocol and an exclamation mark to URLs pointing to attacker-controlled servers."
```

**Pull request merged**